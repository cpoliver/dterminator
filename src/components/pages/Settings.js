import React, { Component } from 'react';
import { View } from 'react-native';

import pageStyles from './pageStyles';
import { Header } from '../common';

class Settings extends Component {
  render() {
    return (
      <View style={pageStyles.view}>
        <Header>Settings</Header>
      </View>
    );
  }
}

export { Settings };
