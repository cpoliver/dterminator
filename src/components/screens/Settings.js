import React from 'react';
import { View } from 'react-native';

import { screenStyles } from '../../styles';
import { Header, Placeholder } from '../common';

const Settings = () => (
  <View style={screenStyles.view}>
    <Header>Settings</Header>
    <Placeholder iconName="settings" />
  </View>
);

export { Settings };
