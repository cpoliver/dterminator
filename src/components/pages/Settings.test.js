import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { Settings } from './Settings';

const render = () => shallow(
  <Settings />
);

describe('the Settings component', () => {
  it('should match the snapshot', () => {
    const component = render();

    expect(toJson(component)).toMatchSnapshot();
  });
});
