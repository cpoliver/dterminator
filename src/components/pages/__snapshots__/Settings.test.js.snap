// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Settings component should match the snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "#f6f7fb",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
>
  <Header>
    Settings
  </Header>
  <Placeholder
    iconName="settings"
  />
</View>
`;
